MAINTAINERS

```diff
@@ -591,7 +598,7 @@ made through a pull request.
 
 	[people.jfrazelle]
 	Name = "Jessie Frazelle"
-	Email = "jess@docker.com"
+	Email = "jess@linux.com"
```

Comment:
Okay can we not digress from the PR, @jfrazelle does what she wants with her emails as long as we trust that the email address she writes here actually goes to her, which hopefully we all do.